src/types/libdef/npm/supertest_vx.x.x.js

```diff
@@ -0,0 +1,45 @@
+// flow-typed signature: cc2e46fdf76da83aa88e0bfaefb15345
```

Comment:
It's a bit sad that there's no libdef for this library.

Reply:
We should probably add some incrementally as we use it.

Reply:
yeah good idea, I'll try to add some.

Reply:
so, I ended up doing a lot of it, and contributed it back to the flow-typed project in https://github.com/flow-typed/flow-typed/pull/3587.